### DISCUSSION TITLE
Can't select a machine type in new code space creation flow

### DISCUSSION THREAD

**Discussion by matthewferry** [community/community#15374] [Codespaces] [answered]
Trying to spin up a new code space from `/codespaces/new` using the new UI form, I am unable to select a machine type other than the default selection. Clicking nor tabbing allows a selection.

https://user-images.githubusercontent.com/895885/164576484-584cbc18-fbcb-42f9-a114-65ad5590d16f.mp4

Edit: looks like an error is happening on selection
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/895885/164579886-5edc27d7-1e81-4fb2-8051-f7a74635d4ba.png">

**Comment by Boogieman2022** [comment]
> Trying to spin up a new code space from `/codespaces/new` using the new UI form, I am unable to select a machine type other than the default selection. Clicking nor tabbing allows a selection.
> 
>  codespace.mech.type.mp4

**Comment by matthewferry** [comment, marked as answer]
This seems to be resolved!
